docker-compose.yaml

```diff
@@ -17,16 +19,36 @@ services:
       - SERVERFULL_RUNTIME_CONNSTATE_NEWCOUNTER=http.server.connstate.new
       - SERVERFULL_RUNTIME_LOGGER_OUTPUT=STDOUT
       - SERVERFULL_RUNTIME_LOGGER_LEVEL=INFO
-      - SERVERFULL_RUNTIME_STATS_OUTPUT=DATADOG
+      - SERVERFULL_RUNTIME_STATS_OUTPUT=NULL
       - SERVERFULL_RUNTIME_STATS_DATADOG_PACKETSIZE=32768
```

Comment:
If you set STATS output to NULL, you should be able to remove all of the `SERVERFULL_RUNTIME_STATS_DATADOG_` settings

Reply:
Ok, cool. Will do!